activities/tool-management/index.md

```diff
@@ -13,11 +13,12 @@ The Foundation for Public Code needs tools to accomplish its mission. This is ho
 
 This guidance covers:
 
-* instructions for users
-* how we manage the tool internally
-* the tool's privacy and GDPR policies
+* Instructions for users
+* How we manage the tool internally
+* The tool's privacy and GDPR policies
+* How and where the tools are managed
 
-Our current tools are:
+## Our current tools are
```

Comment:
This list does not yet include all the tools that are mentioned in the operations handbook - will these be added later?

Do we want to make the distinction here between tools that a) are self managed b) have service providers and c) are entirely managed?

Reply:
The other tools are added and exist, but waiting for the master to be merged to develop before it's visible @lscheske

Reply:
I think the distinction is important, as it also says something about where they weight of the responsibility is for maintenance and such